### PR TITLE
Fix send request to create institution

### PR DESCRIPTION
**Pull request by MaianaB**
<p><b>Feature/Bug description:</b> When a user don't have any institution and send request to create institution. </p>

<p><b>Solution:</b> Fix path to invite request to backend and set institution_requested_key of invite </p>

<p><b>TODO/FIXME:</b> Currently, The Super Institution is 'CIS' but will change to 'Ministério da Saúde', should modify how to verify it</p>

### CODE REVIEW

File: frontend/invites/inviteInstitutionController.js [hunk 1]
```diff
@@ -58,6 +58,36 @@
             }
         };
 
+        inviteInstCtrl.acceptRequest = function acceptRequest(request) {
```
Comment (mayzabeel) [inline frontend/invites/inviteInstitutionController.js]: teste

File: frontend/invites/inviteInstitutionController.js [hunk 2]
```diff
@@ -58,6 +58,36 @@
...
+                MessageService.showToast("Solicitação aceita!");
+                _.remove(inviteInstCtrl.sent_requests, function(req) {
+                    return req == request;
+                });
+            }, function error(response) {
+                MessageService.showToast(response.data.msg);
+            });
+        };
+
+        inviteInstCtrl.rejectRequest = function rejectInvite(event, request){
```
Comment (mayzabeel) [inline frontend/invites/inviteInstitutionController.js]: teste

File: backend/test/institution_request_collection_handler_test.py [hunk 3]
```diff
@@ -12,14 +14,14 @@
...
-    REQUEST_URI = "/api/institutions/requests/institution"
+    REQUEST_URI = "/api/institutions/requests/institution/"
 
     @classmethod
     def setUp(cls):
         """Provide the base for the tests."""
         super(InstitutionRequestCollectionHandlerTest, cls).setUp()
         app = cls.webapp2.WSGIApplication(
-            [(InstitutionRequestCollectionHandlerTest.REQUEST_URI, InstitutionRequestCollectionHandler),
+            [("/api/institutions/requests/institution/(.*)", InstitutionRequestCollectionHandler),
```
Comment (mayzabeel) [inline backend/test/institution_request_collection_handler_test.py]: adicionar assert que verifica se institution_requested é o Complexo

File: backend/utils.py [hunk 4]
```diff
@@ -189,6 +191,14 @@ def check_authorization(self, user, request_key, *args):
         return check_authorization
 
 
+def get_super_institution():
+    """Return Super Institution of system."""
+    # TODO: When change The Super Institution change
```
Comment (mayzabeel) [inline backend/utils.py]: Explicar que atualmente a única instituição com super users é o complexo, mas que futuramente deverá ser o ministério, alterar esse metodo e os testes.

File: backend/utils.py [hunk 5]
```diff
@@ -193,8 +193,8 @@ def check_authorization(self, user, request_key, *args):
 
 def get_super_institution():
     """Return Super Institution of system."""
-    # TODO: When change The Super Institution change
-    # institution name in test of invite
+    # TODO: Currently, The Super Institution is 'CIS' but will change to 'Ministério da Saúde',
+    # should be modified the query and in tests.
```
Comment (mayzabeel) [inline backend/utils.py]: the query and tests

File: backend/test/institution_request_collection_handler_test.py [hunk 6]
```diff
@@ -76,3 +83,13 @@ def initModels(cls):
     cls.other_user.name = 'Other User'
     cls.other_user.email = ['otheruser@test.com']
     cls.other_user.put()
+    # new Institution Address
```
Comment (andreldsa) [inline backend/test/institution_request_collection_handler_test.py]: We have a new approach to initialize mock objects. 

PR: https://github.com/eciis/web/pull/573

File: frontend/test/specs/inviteInstitutionControllerSpec.js [hunk 7]
```diff
@@ -23,19 +23,30 @@
         invites:[]
     };
 
+    var request = {
+        admin_name: 'example',
+        institution_key: splab.key,
```
Comment (andreldsa) [inline frontend/test/specs/inviteInstitutionControllerSpec.js]: Don't use specific names to create object mocks, always use generic names to provide a general environment of tests.

File: frontend/test/specs/inviteInstitutionControllerSpec.js [hunk 8]
```diff
@@ -23,19 +23,30 @@
...
+        institution_requested_key: splab.key,
+        key: '000000',
+        sender: 'admin',
+        status: 'sent',
+    }
+
     var INSTITUTION_SEARCH_URI = '/api/search/institution?value=';
 
     var invite = new Invite({invitee: "mayzabeel@gmail.com", suggestion_institution_name : "New Institution"}, 'institution', splab.key);
```
Comment (andreldsa) [inline frontend/test/specs/inviteInstitutionControllerSpec.js]: Remove specific names.

File: backend/test/institution_request_collection_handler_test.py [hunk 9]
```diff
@@ -79,17 +80,11 @@ def test_post(self, verify_token):
 def initModels(cls):
     """Init the models."""
     # Other user
-    cls.other_user = User()
-    cls.other_user.name = 'Other User'
-    cls.other_user.email = ['otheruser@test.com']
-    cls.other_user.put()
+    cls.other_user = mocks.create_user()
```
Comment (andreldsa) [inline backend/test/institution_request_collection_handler_test.py]: Remove initModels() function and move the creation of the object mocks to the specific test.

File: backend/test/institution_request_collection_handler_test.py [hunk 10]
```diff
@@ -79,17 +80,11 @@ def test_post(self, verify_token):
...
-    cls.address = Address()
-    cls.address.number = '01'
-    cls.address.street = 'street'
+    cls.address = mocks.create_address()
     # new Institution inst requested to be parent of inst test
-    cls.new_inst = Institution()
-    cls.new_inst.name = 'Complexo Industrial da Saude'
-    cls.new_inst.photo_url = 'images/photo.jpg'
+    cls.new_inst = mocks.create_institution()
+    cls.new_inst.name = "Complexo Industrial da Saude"
```
Comment (andreldsa) [inline backend/test/institution_request_collection_handler_test.py]: Don't use specific names within the test.

**Pull request merged**